certified-connectors/IN-D Aadhaar Number Masking/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Key",
+          "description": "The Key for this api",
+          "tooltip": "Provide your Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#dcfffe",
+    "capabilities": [],
+    "publisher": "IN-D by Intain",
+    "stackowner": "IN-D by Intain"

```

Comment:
@gowthamd3 This should be `stackOwner`, not `stackowner`

Reply:
I have done the changes you have mentioned, pls look and let me know if any changes required.